### PR TITLE
Removed unreachable exceptions.

### PR DESCRIPTION
**Pull request by ajeetdsouza**
`ROSInterruptException` is a subclass of `KeyboardInterrupt` and both exceptions are being handled the same way (`pass`). I removed the `ROSInterruptException` handling since it was placed after the `KeyboardInterrupt` and was thus unreachable.

### CODE REVIEW

**Comment by mgrrx** [comment]
LGTM

File: tools/roslaunch/src/roslaunch/remoteprocess.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def _ssh_exec(self, command, address, port, username=None, password=None):
                 err_msg = "Unable to establish ssh connection to [%s%s:%s]: %s"%(username_str, address, port, e)
             except socket.error as e:
                 # #1824
-                if e[0] == 111:
+                if e.args[0] == 111:
```
Comment (dirk-thomas) [inline tools/roslaunch/src/roslaunch/remoteprocess.py:206]: How is this change related to the issue title?

**Review by dirk-thomas — COMMENTED** [review]
Can you please clarify why these exceptions are unreachable.

Please also spend a minimum amount of time when filling and issue / pull request to provide the necessary context.

File: tools/roslaunch/src/roslaunch/remoteprocess.py (hunk 1 above)
Comment (ajeetdsouza) [reply tools/roslaunch/src/roslaunch/remoteprocess.py:206]: I updated the description. Also, changing `e[0]` to `e.args[0]` wasn't really related - I will create separate PRs the next time. Given that I have deleted my fork, is there any way to fix this on my end? Alternatively, you could cherry pick the first commit and ignore the second.

Comment (dirk-thomas) [reply tools/roslaunch/src/roslaunch/remoteprocess.py:206]: Thank you for the additional information.

With the forked repo being delete I am not sure I can cherry-pick individual commits. I will just squash both commits during the merge in this case.
